src/web/settings.py

```diff
@@ -162,7 +161,7 @@ def join_path(self, template: str, parent: str) -> str:
 
 template_dirs: list[Path] = [
     BASE_DIR / 'src/web/templates',
-    *[path for path in plugin_manager.template_paths],
+    *[path for path in plugin_manager.templates_paths if path],
```

Comment:
```suggestion
    *[path for path in plugin_manager.templates_paths],
```
path never is not set